code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,96 @@
+	desc = "Link to the worlds beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	var/datum/mind/target
+
+/obj/item/living_heart/attack_self(mob/user)
+	. = ..()
+	if(!isecultist(user))
+		return
+	if(!target)
+		return
+	var/dist = get_dist(user,target)
+
+	switch(dist)
+		if(0 to 5)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is near you</span>")
+		if(6 to 15)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is somewhere in your vicinty</span>")
+		if(16 to 64)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is far away from you</span>")
+		else
+			to_chat(user,"<span class='warning'>[target.current.real_name] is beyond our reach</span>")
+
+	if(target.current.stat == DEAD)
+		to_chat(user,"<span class='warning'>[target.current.real_name] is dead. Bring him onto the transmutation rune!</span>")
+
+/obj/item/melee/sickly_blade
+	name = "Sickly blade"
+	desc = "Twisted sickle with an ornamental eye. The eyes looks at you."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eldritch_blade"
+	item_state = "eldritch_blade"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	sharpness = IS_SHARP
+	w_class = WEIGHT_CLASS_BULKY
+	force = 15
+	throwforce = 10
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "rended")
+
+/obj/item/melee/sickly_blade/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	var/datum/antagonist/ecult/cultie = user.mind.has_antag_datum(/datum/antagonist/ecult)
+	if(!cultie)
+		return
+	for(var/X in cultie.get_all_knowledge())
+		var/datum/eldritch_knowledge/EK = X
+		EK.eldritch_blade_act(target,user,proximity_flag,click_parameters)
+
+/obj/item/melee/sickly_blade/rust
+	name = "Rusted Blade"
+
+/obj/item/melee/sickly_blade/ash
+	name = "Ashen Blade"
+
+/obj/item/melee/sickly_blade/flesh
+	name = "Flesh Blade"
+
+/obj/item/clothing/neck/eldritch_amulet
+	name = "Warm Eldritch Medallion"
+	desc = "Eldritch medallion that let's you see."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eye_medalion"
+	item_state = ""	//no inhands
+	w_class = WEIGHT_CLASS_SMALL
+	trinket_flag = SEE_MOBS
+	trinket_lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+
+/obj/item/clothing/neck/eldritch_amulet/piercing
+	name = "Piercing Eldritch Medallion"
+	trinket_flag = SEE_TURFS|SEE_MOBS|SEE_OBJS
+
+/obj/item/clothing/head/hooded/cult_hoodie/eldritch
+	name = "ominous hood"
+	icon_state = "eldritch"
+	desc = "A torn, dust-caked hood. Strange eyes line the inside."
+
+/obj/item/clothing/suit/hooded/cultrobes/eldritch
+	name = "ominous armor"
+	desc = "A ragged, dusty set of robes. Strange eyes line the inside."
+	icon_state = "eldritch_armor"
+	item_state = "eldritch_armor"
+	allowed = list(/obj/item/melee/sickly_blade, /obj/item/forbidden_book)
```

Comment:
make sure that a sickly blade can actually fit in the suit storage slot of a suit of ominous armor (as it's a bulky item)

Reply:
good catch